matomo/content.md

```diff
@@ -0,0 +1,60 @@
+# Matomo (formerly Piwik)
+
+[![Build Status](https://travis-ci.org/matomo-org/docker.svg?branch=master)](https://travis-ci.org/matomo-org/docker)
+[Matomo](https://matomo.org/) (formerly Piwik) is the leading open-source analytics platform that gives you more than just powerful analytics:
+
+- Free open-source software
+- 100% data ownership
+- User privacy protection
+- User-centric insights
+- Customisable and extensible
+
+%%LOGO%%
+
+## Usage
+
+In keeping with a 'pure' micro-services approach, this image runs a Matomo service only (in the form of FastCGI). Because of that it **must** be used with companion containers which provide a database for data storage and HTTP to FastCGI proxy/translation services for the user interface.
+
+## Runtime
+
+You can run the Matomo container and service like so:
+
+```bash
+docker run -d --link some-mysql:db matomo
+```
+
+This assumes you've already launched a suitable MySQL or MariaDB database container.
+
+You'll now need to use a suitable reverse proxy to access the user interface; which is available on TCP port 9000. Nginx provides the necessary functions for translation between HTTP and FastCGI.
+
+## Matomo Installation
+
+Once you're up and running, you'll arrive at the configuration wizard page. If you're using the compose file, at the `Database Setup` step, please enter the following:
+
+- Database Server: `db`
+- Login: `root`
+- Password: MYSQL_ROOT_PASSWORD
+- Database Name: piwik (or you can choose)
+
+And leave the rest as default.
+
+Then you can continue the installation with the super user.
+
+## Docker-composer examples and log import instructions
+
+A minimal set-up using docker-compose is available in the [.examples folder](.examples/docker-compose.yml), a more complete [example can be found at IndieHosters/piwik](https://github.com/libresh/compose-matomo/blob/master/docker-compose.yml).
```

Comment:
broken link: `.examples/docker-compose.yml`.  Perhaps you could use `%%GITREPO%%/blob/master/.examples/docker-compose.yml`.

It looks like that compose file could use an update to `matomo` on the image name as well as the IndieHosters link text